src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -113,6 +126,17 @@ public override bool Execute()
         GatherAotSourcesObjects(aotSources, aotObjects, extraSources, linkerArgs);
         GatherLinkerArgs(linkerArgs);
 
+        File.WriteAllText(Path.Combine(OutputDirectory, "shared_library_log.h"),
```

Comment:
I think we should just name this library_builder.h and be done.

Reply:
or use - as delimitator since we use that for other header files, so library-builder.h